selfdrive/car/body/bodycan.py

```diff
@@ -1,8 +1,18 @@
-def create_control(packer, torque_l, torque_r):
+from selfdrive.car import crc8_pedal
+
+
+def create_control(packer, torque_l, torque_r, idx):
   can_bus = 0
 
   values = {
     "TORQUE_L": torque_l,
     "TORQUE_R": torque_r,
+    "COUNTER": idx & 0xF,
```

Comment:
packer should set this

Reply:
Why do we do the same way for pedal then? in create_gas_interceptor_command()

Reply:
Also because of the signal name?

Reply:
not sure, sounds likely though